cmd/config.go

```diff
@@ -46,29 +46,41 @@ var configurationSetCmd = &cobra.Command{
 	Short: "Set a value in odo config file",
 	Long: `Set an individual value in the Odo configuration file 
 Available Parameters:
-UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+UpdateNotification - Controls if an update notification is shown or not (true or false)
+Timeout            - Timeout (in seconds) for openshift server connection check`,
```

Comment:
This isn't the same output as https://github.com/redhat-developer/odo/pull/757/files#diff-31f9f805fb3e45c114bdaa74f9447759R22

Please update both. Also openshift = OpenShift!